src/dbos-executor.ts

```diff
@@ -258,7 +266,7 @@ export class DBOSExecutor {
     for (const ro of registeredClassOperations) {
       if (ro.workflowConfig) {
         const wf = ro.registeredFunction as Workflow<any, any>;
-        this.#registerWorkflow(wf, ro.workflowConfig);
+        this.#registerWorkflow(wf, {registrationObject: ro, ...ro.workflowConfig});
```

Comment:
Same comment as the other one: can we simply pass `schedulerConfig` here.

Reply:
I still think for now we should pass in `schedulerConfig` only. Because it's confusing to pass both `ro` and `ro.workflowConfig` which is redundant.